cms/djangoapps/contentstore/proctoring.py

```diff
@@ -39,7 +39,10 @@ def register_special_exams(course_key):
         # if feature is not enabled then do a quick exit
         return
 
-    course = modulestore().get_course(course_key)
+    try:
+        course = modulestore().get_course(course_key)
```

Comment:
Yeah. You are right. If a course is `None` then `AttributeError ` would be raised. So I just checked the course before using it, that's why I have added try, catch.